### ISSUE TITLE
Realtime effects accessibility: there is no indication for screen reader users which controls are for master effects

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#6456]
### Steps to reproduce

With the NVDA screen reader running:
1. Press E to open the realtime effects pane.
2. Tab through the controls in the pane.

### Expected behavior

There should be an indication which controls are for master  and which are for the focused track

### Actual behavior

There is no indication which controls are for master and which are for the focused track.

### Audacity Version

other

### Operating system

Windows

### Additional context

Audacity version: https://github.com/audacity/audacity/commit/e5267d86470f8f74c040527f994adf51502124a1

**Comment by DavidBailes** [comment]
@dozzzzer . I think this is a serious accessibility bug. Please could you assign an priority to it?

**Comment by DavidBailes** [comment]
@LWinterberg . Please can this be added to 3.6, rather than 3.6.1. This issue makes this it difficult for screen readers to use this new feature.
